TelCo.ColorCoder/DefineColorLists.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Diagnostics;
```

Comment:
Partial classes:
When a partial class is used, it comes with a cost of readability
Additional effort is required to decipher the partial class across multiple files.
Partial classes are mainly used 
- to splitting auto-generated code from main code.
- To de-clutter code for example for separating boiler plate code from main code
- In places where a class implements a lot of interfaces. Partial class can be used 1 file, 1 partial class for 1 interface implementation.

In cases other than the ones mentioned above, instead of partial class a new class can be created and aggregated to the main class.